### PR TITLE
Expanded on the documentation

### PR DESCRIPTION
**Pull request by sergeifilippov**
Added example of setting up a multi server deployment with custom property definitions per environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +232,39 @@ shipit.log('hello %s', 'world');
...
+
+You can overwrite all default variables defined as part of the `default` object.
+
+```js
+module.exports = function (shipit) {
+  shipit.initConfig({
+    staging: {
+      servers: 'staging.myproject.com',
+      workspace: '/home/vagrant/website'
+      branch: "dev"
```
Comment (neoziro) [inline README.md]: single quotes

File: README.md [hunk 2]
```diff
@@ -232,6 +232,39 @@ shipit.log('hello %s', 'world');
...
+    },
+    production: {
+      servers: [{
+        host: 'app1.myproject.com',
+        user: 'john',
+      }, {
+        host: 'app2.myproject.com',
+        user: 'rob',
+      }],
+      branch: "production",
```
Comment (neoziro) [inline README.md]: single quotes

**Comment by sergeifilippov** [comment]
Sorry, do you want me to make every thing added to be Single Quotes or remove Single Quotes and make it all Double Quotes?

**Comment by neoziro** [comment]
Single quotes everywhere please.

**Pull request merged**